Protocol.md

```diff
@@ -154,7 +154,7 @@ ReactiveSocket frames begin with a ReactiveSocket Frame Header. The general layo
 ```
 
 * __Stream ID__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer representing the stream Identifier for this frame or 0 to indicate the entire connection.
-  * Transport protocols that include demultiplexing, such as HTTP/2, MAY omit the Stream ID field if all parties agree. The means of negotiation and agreement is left to the transport protocol. 
+  * Transport protocols that include demultiplexing, such as HTTP/2, MAY omit the Stream ID field if all parties agree. The means of negotiation and agreement is left to the transport protocol.
```

Comment:
When I updated the spec a couple weeks back to clarify HTTP/2, I adjusted to use "HTTP/2 stream" and not just "HTTP/2". We could add a section explaining what this is about. 

It remains somewhat ambiguous right now though.